helm/ngrok-operator/values.yaml

```diff
@@ -56,6 +56,33 @@ apiURL: "" # default to ngrok-api-go SDK's default
 ##
 ngrokMetadata: {}
 
+## @param ngrokRootCA The ngrok root CA certificate to use for verifyng self-signed TLS certs from ngrok
+## Note: This is temporarilyt vendored and will be removed in a future release.
+ngrokRootCA: | # ngrok intermediate CA
```

Comment:
I think this certificate is specific to bindings and is an intermediate CA, not a root. Do we want to nest this under bindings since it only applies to bindings?

Reply:
Sure, will adjust after techstrong demo 👍